utils/api.py

```diff
@@ -245,7 +248,8 @@ def find_by(self, **params):
     def _find_by_sqlfilter(self, **params):
         search_query = []
         for key, value in params.iteritems():
-            search_query.append("{} = {}".format(key, repr(str(value))))
+            # We may need to escape some chars here!
+            search_query.append("{} = '{}'".format(key, value))
```

Comment:
But this file is going away and _find_by_sqlfilter shall not be invoked in the recent cfme versions.